assets/javascripts/modules/analytics/omniture.js

```diff
@@ -20,6 +20,8 @@ define(function () {
         s.prop47 = 'UK';
         s_code = s.t();
 
+        checkoutOmniture.init(s);
```

Comment:
Is there a way to write this so the checkout code is initialised elsewhere? Or does is need to be done here for Omniture?